config/test.exs

```diff
@@ -12,6 +12,8 @@ config :logger, level: :warn
 # Disable authentication mode during test
 config :livebook, :authentication_mode, :disabled
 
+config :livebook, :data_path, "tmp/livebook_data/test"
```

Comment:
@josevalim we need relative paths here, so I added expand directly in the call, let me know if you have any better idea for this :)